docker-compose.yml

```diff
@@ -25,4 +25,14 @@ services:
     build: ./
     image: "${DOCKER_IMAGE}-test:${TAG:-latest}"
     depends_on:
-      - standalone-chrome
\ No newline at end of file
+      - standalone-chrome
```

Comment:
We can remove the test-ci service if I'm not mistaken

Reply:
yeah, i can do that